### ISSUE TITLE
[FEATURE] Kill Aura Only crits option

### ISSUE THREAD

**Issue by szymcelo** [CCBlueX/LiquidBounce#2593]
### LiquidBounce Branch

Nextgen

### Describe your feature request.

Add option for kill aura to only perform attacks that will 100% be critical.
And maybe change optimise for criticals to prefer crits? (If both disabled then kill aura hit just after cooldown)

### Additional context

Not needed

**Comment by superblaubeere27** [comment]
Which version do you use? KillAura already offers an option `OptimizeForCriticals` to prefer crits.

Are you using it?

**Comment by szymcelo** [comment]
Yes I’m using it, but for certain modes like netherite, I would still prefer to have 100% crits, rather than crit, normal, crit
